.circleci/scripts/cherry-picker.sh

```diff
@@ -165,11 +165,12 @@ if echo "$resp" | jq -e '.items[].labels[] | select(.name | contains("docs-cherr
     fi
 fi
 
+git config --local user.email "github-team-consul-core@hashicorp.com"
+git config --local user.name "hc-github-team-consul-core"
```

Comment:
This is unrelated, but I noticed while reading it over that we were setting this in a loop, and I believe we only need to set this once.